docs/installation/ubuntulinux.md

```diff
@@ -94,12 +94,31 @@ packages from the new repository:
 
 ### Prerequisites by Ubuntu Version
 
-The following Ubuntu versions have no additional prerequisites:
-
 - Ubuntu Wily 15.10
 - Ubuntu Vivid 15.04
 - Ubuntu Trusty 14.04 (LTS)
 
+For Ubuntu Trusty, Vivid, and Wily, it's recommended to install the
+`linux-image-extra` kernel package. The `linux-image-extra` package
+allows you use the `aufs` storage driver.
+
+To install the `linux-image-extra` package for your kernel version:
+
+1. Open a terminal on your Ubuntu host.
+
+2. Update your package manager.
+
+        $ sudo apt-get update
+
+3. Install the recommended package.
+
+        $ sudo apt-get install linux-image-extra-$(uname -r)
```

Comment:
This will install that






On Thu, Nov 5, 2015 at 7:53 AM -0800, "Sebastiaan van Stijn" <notifications@github.com> wrote:












In docs/installation/ubuntulinux.md:
>  - Ubuntu Wily 15.10
>  - Ubuntu Vivid 15.04
>  - Ubuntu Trusty 14.04 (LTS)
>  
> +For Ubuntu Trusty, Vivid, and Wily, it's recommended to install the
> +`linux-image-extra` kernel package. The `linux-image-extra` package
> +allows you use the `aufs` storage driver.
> +
> +To install the `linux-image-extra` package for your kernel version:
> +
> +1. Open a terminal on your Ubuntu host.
> +
> +2. Update your package manager.
> +
> +        $ sudo apt-get update
> +
> +3. Install the recommended package.
> +
> +        $ sudo apt-get install linux-image-extra-$(uname -r)



hm, yeah let me check, iirc that was not for all kernel versions, but I might be mistaken here



—
Reply to this email directly or view it on GitHub.